host_core/native/hostcore_wasmcloud_native/src/par.rs

```diff
@@ -23,9 +22,24 @@ pub(crate) fn get_vendor(par: &ProviderArchive) -> Result<String, Error> {
     }
 }
 
-pub(crate) fn extract_claims(par: &ProviderArchive) -> Result<Claims, Error> {
+pub(crate) fn extract_claims(par: &ProviderArchive) -> Result<crate::Claims, Error> {
     par.claims()
-        .map(Claims::from)
+        .map(|c| {
```

Comment:
so this bit of copy-paste didn't work if you tried to call the function `provider_claims_to_crate_claims`?

Reply:
nope. still complains about type mismatch

Reply:
What was the exact error? I think I can help with this

Reply:
It was complaining that there was no implementation of `From<...>` for the thing, and then told me that there _was_ an implementation called the same thing.

The whole thing felt very much like it was caused by overlapping or conflicting names, but I could be wrong.